### PR TITLE
Don't show interventions in Refer task list until they're chosen in Find

### PR DESCRIPTION
**Pull request by lawrence-forooghian**
# What's new

Now that the list of interventions isn't known until the Find task has been completed, we now only show the intervention details tasks in Refer after Find has been done. We turn the intervention details into a single task with subsections, to allow the task list to maintain a consistent numbering even before the number of interventions is known.

# Screenshots

## Before finding interventions

image

## After finding interventions

image

### CODE REVIEW

File: app/views/sprint-6/book-and-manage/make-a-referral/referral-task-list.html [hunk 1]
```diff
@@ -178,169 +178,173 @@ <h2 class="moj-task-list__section">
...
+            {% endif %}
+            <h2 class="moj-task-list__section">
+              <span class="moj-task-list__section-number">5.
+              </span>
+              Accommodation referral
+            </h2>
+            <ul class="moj-task-list__items">
+              <li class="moj-task-list__item">
+                {% if data["needs-and-requirements-completed"] %}
+                  <a class="moj-task-list__task-name" href="accommodation-service/select-sentence">Select relevant sentence</a>
```
Comment (Gweaton) [inline app/views/sprint-6/book-and-manage/make-a-referral/referral-task-list.html:216]: For the social inclusion sentence, we say "Select relevant sentence for the social inclusion service referral", do we want to be consistent here (and say "for the accommodation service referral")?

File: app/views/sprint-6/book-and-manage/make-a-referral/referral-task-list.html [hunk 2]
```diff
@@ -178,169 +178,173 @@ <h2 class="moj-task-list__section">
...
+                    Select required complexity level
+                  </span>
+                {% endif %}
+              </li>
+              <li class="moj-task-list__item">
+                {% if data["accommodation-desired-outcomes"] %}
+                  <a class="moj-task-list__task-name" href="accommodation-service/completion-date">What date does the accommodation service need to be completed by?</a>
+                {% else %}
+                  <span class="app-task-list__task-name">
+                    What date does the accommodation service need to be completed by?
```
Comment (Gweaton) [inline app/views/sprint-6/book-and-manage/make-a-referral/referral-task-list.html:246]: This is also inconsistent with the social inclusion one (which just says "Enter completion date")

**Review by Gweaton — APPROVED** [review]

File: app/views/sprint-6/book-and-manage/make-a-referral/referral-task-list.html (hunk 1 above)
Comment (lawrence-forooghian) [reply app/views/sprint-6/book-and-manage/make-a-referral/referral-task-list.html]: Good shout, changed it.

File: app/views/sprint-6/book-and-manage/make-a-referral/referral-task-list.html (hunk 2 above)
Comment (lawrence-forooghian) [reply app/views/sprint-6/book-and-manage/make-a-referral/referral-task-list.html:246]: 👍 changed

**Pull request merged**